src/modules/terminalfont.c

```diff
@@ -69,6 +67,51 @@ static void printXFCE4Terminal(FFinstance* instance)
     printTerminalFont(instance, font);
 }
 
+static void printTilixTerminal(FFinstance* instance)
+{
+    static const char* fontName = NULL;
+    static const char* defaultProfile = NULL;
+    static FFstrbuf key;
+    static FFvariant res;
+    res.strValue = NULL;
+    ffStrbufInitAS(&key, 64, "/com/gexperts/Tilix/profiles/");
+
+    defaultProfile = (ffSettingsGetGsettings(instance, "com.gexperts.Tilix.ProfilesList", NULL, "default", FF_VARIANT_TYPE_STRING)).strValue;
+
+    if (!defaultProfile)
+    {
+        ffPrintError(instance, FF_TERMFONT_MODULE_NAME, 0, &instance->config.termFontKey, &instance->config.termFontFormat, FF_TERMFONT_NUM_FORMAT_ARGS, "Couldn't find \"Default\" profile in Tilix settings");
+        ffStrbufDestroy(&key);
```

Comment:
Key is never needed at this point, so we destroy it unused. Wouldn't it make more sense to declare it after the `if`?